README.md

```diff
@@ -1 +1,66 @@
 # wizard-additional-playing-cards
+
+## Tornado | 14 | blau
+
+- Nach dem Stich wird in die andere Richtung gespielt
+- Bis zum Ende der Runde
+
+## Katapult | 14 | rot
+
+- Alle ziehen eine Karte
+
+## Frosch | 14 | grün
+
+- Alle müssen eine Karte auf den Stapel legen
```

Comment:
- [ ] Was macht man in der Letzten Runde, wenn man nur noch eine Karte hat?
- [ ] Wann muss man das ansagen/ spielen? Am Anfang einer Runde, oder während einer Runde?